cmip6_downscaling/methods/common/tasks.py

```diff
@@ -161,6 +162,25 @@ def pyramid(
     return target
 
 
+@task
+def regrid(source_path: UPath, target_grid_path: UPath) -> UPath:
+
+    target = intermediate_dir / "regrid" / source_path / target_grid_path
+
+    if use_cache and (target / '.zmetadata').exists():
+        print(f'found existing target: {target}')
+        return target
+
+    source_ds = xr.open_zarr(source_path)
+    target_grid_ds = xr.open_zarr(target_grid_path).isel(time=0)
```

Comment:
fwiw, I don't think the `isel` bit here is needed. Probably doesn't hurt but not strictly needed.

Reply:
Ah good to know! I'll remove it for clarity.